### DISCUSSION TITLE
Suggestion

### DISCUSSION THREAD

**Discussion by LilSefGaming** [ScratchAddons/ScratchAddons#4907] [Ideas]
How about your own featured projects?!
Everybody who is using scratch addons can submit their own projects to github ScratchAddons, and whoever they think about the top five projects should be featured, and whoever enables that feature on the Addons page can see a whole new featured category called "ScratchAddons Featured Projects".
Wouldn't that be GREAT?!

**Comment by DNin01** [comment]
It _would_ be fun, but a lot of time is already being spent developing Scratch Addons. With such an addon, suddenly having to review probably hundreds of projects overtime and picking ones to feature would take time away from development and slow down updates unless we were to find some collaborators to maintain this feature specifically. But mainly, it's also beside the point of Scratch Addons, which is to help you be more productive and enhance the user experience on Scratch.

But there's nothing stopping this from being a thing elsewhere. Maybe someone will make a new browser extension or community for this.
